src/app/public/modules/colorpicker/colorpicker.component.scss

```diff
@@ -303,11 +326,19 @@
     bottom: 0;
     right: 2px;
     font-size: 10px;
+    &.disabled {
+      color: $sky-theme-modern-font-deemphasized-color;
+    }
   }
 
   .sky-colorpicker-reset-button-modern {
     width: 40px;
     height: 40px;
+    &.disabled {
+      background-color: $sky-theme-modern-border-color-neutral-light;
+      border: 1px solid $sky-theme-modern-border-color-neutral-medium;
```

Comment:
it looks like this border gets doubled up with an inset box-shadow set somewhere else so it ends up being 2px. might not need this if buttons already have the inset box-shadow.

Reply:
gotcha. ive set the border to be 0px, so that only the box-sadow remains. (fixed in last commit)